benchopt/runner.py

```diff
@@ -203,7 +203,7 @@ def run_one_solver(benchmark, dataset, objective, solver, n_repetitions,
             idx_rep=rep,
             stopping_strategy=stopping_strategy.capitalize(),
             obj_description=obj_description,
-            solver_description=solver.__doc__ or "",
+            solver_description=(solver.__doc__ or "").replace("    ", ""),
```

Comment:
looks :ok_hand: @mathurinm !

Reply:
this will mess up with list for instance no?
why not use the [dedent function](https://www.oreilly.com/library/view/mastering-python-scripting/9781789133226/38c16e80-ef5e-44fe-8b12-241c9d6a0e58.xhtml#:~:text=of%20O'Reilly.-,The%20dedent()%20function,every%20line%20of%20your%20text.&text=text%20is%20the%20text%20to%20dedent%20.)?